src/render/md.jl

```diff
@@ -1,17 +1,43 @@
 ## Docs-specific rendering ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
 
-function writemime(io::IO, mime::MIME"text/md", docs::Docs{:md})
+function writemime(io::IO, mime::MIME"text/md", docs::Docs{:md}, mdstyle::Dict{ASCIIString, ASCIIString} = DEFAULT_MDSTYLE)
     println(io, docs.data)
 end
 
 ## General markdown rendering ------------------------–––––––––––––––––––––––––––––––––––
 
-function save(file::String, mime::MIME"text/md", doc::Metadata; mathjax = false, include_internal = true)
+const DEFAULT_MDSTYLE = Dict{ASCIIString, ASCIIString}([
+  ("header"         , "#"),
+  ("objname"        , "####"),
+  ("meta"           , "**"),
+  ("exported"       , "##"),
+  ("internal"       , "##"),
+])
+
+const HTAGS = ["#", "##", "###", "####", "#####", "######"]
+const STYLETAGS = ["", "*", "**"]
+
+print_help(io::IO, cv::ASCIIString, item) = cv in HTAGS ? println(io, "$cv $item") : println(io, cv, item, cv)
+
+function validate(mdstyle::Dict{ASCIIString, ASCIIString})
+    reg_keys = ["header", "objname", "meta", "exported", "internal"]
+    length(keys(mdstyle)) != length(reg_keys) && error(
+            "`mdstyle` expected number of keys: $(length(reg_keys))  Got: $(length(keys(mdstyle)))")
+    vaild_tags = vcat(HTAGS,STYLETAGS)
+    for (k, v) in mdstyle
+        !(k in reg_keys) && error("Invalid mdstyle key:  `$k`. Valid keys: [$(join(reg_keys, ", "))].")
+        !(v in vaild_tags) && error("Invalid mdstyle value:  `$v`. Valid values: [$(join(vaild_tags, ", "))].")
+    end
```

Comment:
Will change that.
I just started with julia and decided for the beginning to use only `&&`

*Both && and || associate to the right, `but && has higher precedence` than || does* (http://julia.readthedocs.org/en/latest/manual/control-flow/#short-circuit-evaluation)